test/syncenginetestutils.cpp

```diff
@@ -128,12 +128,6 @@ void DiskFileModifier::setE2EE([[maybe_unused]] const QString &relativePath, [[m
 {
 }
 
-QFile DiskFileModifier::find(const QString &relativePath) const
-{
-    const auto path = _rootDir.filePath(relativePath);
-    return QFile(path);
-}
```

Comment:
why did this get removed?

(fwiw that's fine by me as long as the other tests don't depend on that specific method)